src/cephadm/tests/test_util_funcs.py

```diff
@@ -414,3 +415,130 @@ def test_check_unit(enabled_out, active_out, expected):
         )
         enabled, state, installed = _cephadm.check_unit(ctx, "foobar")
     assert (enabled, state, installed) == expected
+
+
+class FakeEnabler:
+    def __init__(self, should_be_called):
+        self._should_be_called = should_be_called
+        self._services = []
+
+    def enable_service(self, service):
+        self._services.append(service)
+
+    def check_expected(self):
+        if not self._should_be_called:
+            assert not self._services
+            return
+        assert len(self._services) >= 7
```

Comment:
just curios about why 7 :)

Reply:
I assume this is because there are currently 7 options we have listed for a time service
```
units = [
        'chrony.service',  # 18.04 (at least)
        'chronyd.service',  # el / opensuse
        'systemd-timesyncd.service',
        'ntpd.service',  # el7 (at least)
        'ntp.service',  # 18.04 (at least)
        'ntpsec.service',  # 20.04 (at least) / buster
        'openntpd.service',  # ubuntu / debian
    ]
```
so `>= 7` means it tries all the ones currently (which is what it should do) but we don't have to update the test if we add more.

Reply:
Exactly right :-)

Reply:
I see, in that case probably a comment could help to clarify the thing in this case :)

Reply:
Yes, thanks! Updated.